digital-twin-model/src/sdv_v1.rs

```diff
@@ -2,30 +2,52 @@
 // Licensed under the MIT license.
 // SPDX-License-Identifier: MIT
 
-// Note: In the future this code could be auto-generated from a DTDL spec.
+// Note: In the future this code should be generated from a DTDL spec.
 
-pub mod vehicle {
-    pub mod cabin {
-        pub mod hvac {
-            pub mod ambient_air_temperature {
-                pub const ID: &str = "dtmi:sdv:Vehicle:Cabin:HVAC:AmbientAirTemperature;1";
-            }
-            pub mod is_air_conditioning_active {
-                pub const ID: &str = "dtmi:sdv:Vehicle:Cabin:HVAC:IsAirConditioningActive;1";
-            }
-        }
-        pub mod infotainment {
-            pub mod hmi {
-                pub mod show_notification {
-                    pub const ID: &str =
-                        "dtmi:sdv:Vehicle:Cabin:Infotainment:HMI:ShowNotification;1";
-                }
-            }
-        }
+pub mod airbag_seat_massager {
+    pub mod massage_airbags {
+        pub const ID: &str = "dtmi::sdv::airbag_seat_massager::massage_airbags";
```

Comment:
The ID for massage airbags in your DTDL is `dtmi:sdv:AirbagSeatMassager:MassageAirbags;1`. 

Should this pub const ID match exactly to your DTDL?

Reply:
Good catch.  Fixed.